src/Deal/Interfaces/IDealHubSpotEntity.cs

```diff
@@ -8,7 +8,7 @@ public interface IDealHubSpotEntity : IHubSpotEntity
         string Name { get; set; }
         string Stage { get; set; }
         string Pipeline { get; set; }
-        string OwnerId { get; set; }
+        long? OwnerId { get; set; }
```

Comment:
**question**

Are we sure that we can _always_ materialize the ownerIds as longs ?

Reply:
I quess we can, just looked at their documentation.